src/org/traccar/protocol/OsmAndProtocolDecoder.java

```diff
@@ -128,6 +128,9 @@ protected Object decode(
                 case "batt":
                     position.set(Position.KEY_BATTERY_LEVEL, Double.parseDouble(value));
                     break;
+                case "driverUniqueId":
```

Comment:
You are right. If driver id is digits-only, then it would try to save it as Double.